From e63ef7f541bec48e8070e586048fbd717dc3f379 Mon Sep 17 00:00:00 2001
From: HeYue <yue.he@intel.com>
Date: Mon, 15 Sep 2025 12:01:00 +0800
Subject: [PATCH] Add new process name of HWC service

The new version of drm-hwcomposer changes the process name of HWC
service. Add it to sepolicy.

Tracked-On: OAM-134057
Signed-off-by: Weifeng Liu <weifeng.liu@intel.com>
---
 graphics/composer3/file_contexts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/graphics/composer3/file_contexts b/graphics/composer3/file_contexts
index 9ca63c0131dc..29c8f184ef57 100644
--- a/graphics/composer3/file_contexts
+++ b/graphics/composer3/file_contexts
@@ -1,2 +1,3 @@
 /(vendor|system/vendor)/bin/hw/android\.hardware\.graphics\.composer3-service\.intel u:object_r:hal_graphics_composer_default_exec:s0
+/(vendor|system/vendor)/bin/hw/android\.hardware\.composer\.hwc3-service\.drm u:object_r:hal_graphics_composer_default_exec:s0
 
-- 
2.43.2

